servicetalk-tcp-netty-internal/src/main/java/io/servicetalk/tcp/netty/internal/TcpServerConfig.java

```diff
@@ -95,9 +126,10 @@ public void transportObserver(final TransportObserver transportObserver) {
      * found the corresponding {@link ServerSslConfig} is used.
      * @return {@code this}
      */
-    public TcpServerConfig sslConfig(ServerSslConfig defaultSslConfig, Map<String, ServerSslConfig> sniConfig) {
```

Comment:
This is a public API change. Consider deprecating pre-existing `sslConfig(SslConfig)` method before adding `sslConfig(SslConfig, boolean)`